### PR TITLE
Add components for mainmenu page and import components to storybook

### PR DESCRIPTION
**Pull request by atorczynski**
Changes contain minor modifications to structure to improve readability.

### CODE REVIEW

File: client/src/components/Input.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  padding: 3px;
+  transform: rotate(-45deg);
+  -webkit-transform: rotate(-45deg);
+`;
+
+const Input = styled.input`
+  margin-bottom: 50px;
+  border: none;
+  margin: 0;
+  border-bottom: 1px solid rgba(211, 211, 211, 1);
```
Comment (manuelwecker) [inline client/src/components/Input.js]: You wanna work with RGB color definitition? Your choice of course

File: client/src/components/MainOverview.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const OverviewWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+`;
+
+const Seperator = styled.hr`
+  width: 273px;
```
Comment (manuelwecker) [inline client/src/components/MainOverview.js]: Maybe you want to stick to a grid an operate with even numbers (e.g. 280px)?

File: client/src/components/MainOverview.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  height: 0px;
+  background: transparent;
+  border: 1px solid #707070;
+  opacity: 0.17;
```
Comment (manuelwecker) [inline client/src/components/MainOverview.js]: Same here: from a design perspective I would choose 0.15 oder 0.20. Just an idea ;-)

File: client/src/components/Input.js (hunk 1 above)
Comment (atorczynski) [reply client/src/components/Input.js]: Thx for your feedback. Using rgba enables the possibility to change the opacity on here.

**Pull request merged**